### ISSUE TITLE
File structure changes when publishing

### ISSUE THREAD

**Issue by rodrigopina360** [svishnevsky/PDFtoPrinter#48]
Hello! I have come across a problem I've tried to fix to avail. I have a project with your nuget, and when built, copies it's executable to the root of the project, which is the expected behavior, but when published, the executable is moved to a newly created file structure "packages/PDFtoPrinter.1.5.0/build". I've found no way to override this behavior, so that the executable is placed at the root of the project. 
As I have multiple other nugets installed, and none display this issue, I believe it might be a problem with this specific nuget, but would like some confirmation, and if so, what alternatives could be achieved to get working functionality when published.

**Comment by svishnevsky** [comment]
Hi @Pinossaur 

Thanks for reporting the issue. Could you please provide the version of .net framework you use and type of the application (console, web. desktop etc).

**Comment by rodrigopina360** [comment]
I'm using .net framework 4.6.1 on a WPF desktop application

**Comment by svishnevsky** [comment]
Thanks @Pinossaur !

Just to confirm, do you publish to a local folder? Is the package installed to the WPF project itself or some library?

I've run tests and "setup.exe" copies files properly to me:
![image](https://user-images.githubusercontent.com/2376915/228758494-6884b14c-1f11-4846-9e67-63368c6766c6.png)

So I need a little bit more information from you to reproduce your case.

**Comment by rodrigopina360** [comment]
Yeah, I indeed publish to a local folder, and have the package installed to the project itself. Besides that the only thing to note is that ClickOnce is used.

**Comment by svishnevsky** [comment]
Sorry @Pinossaur , still can't reproduce your issue. Could you please share the "csproj" file of yours with me? This might help to reproduce the issue.
